File: ex-14/ex-14_java/Cor.java
public class Cor {
    public int r;
    public int g;
    public int b;
    public int a;

    public Cor(int r, int g, int b) {
        this.r = r;
        this.g = g;
        this.b = b;
        this.a = 255;
    }

    public Cor(int r, int g, int b, int a) {
        this.r = r;
        this.g = g;
        this.b = b;
        this.a = a;
    }

    public static Cor Black() {
        return new Cor(0, 0, 0);
    }

    public static Cor Yellow() {
        return new Cor(255, 255, 0);
    }

    public static Cor Cyan() {
        return new Cor(0, 255, 255);
    }
}

File: ex-14/ex-14_python/cor.py
class Cor:
    def __init__(self, r, g, b, a=255):
        self.r = r
        self.g = g
        self.b = b
        self.a = a

    @staticmethod
    def Black():
        return Cor(0, 0, 0)

    @staticmethod
    def Yellow():
        return Cor(255, 255, 0)

    @staticmethod
    def Cyan():
        return Cor(0, 255, 255)
